packages/carbon-graphs/tests/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -367,6 +367,88 @@ describe('Paired Result - Load', () => {
       );
       expect(point.length).toBe(0);
     });
+    it('does not update x axis range if allow calibration is disabled', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
```

Comment:
nit: rename `disableCalibrationInput` to `graphConfig` and
`disableCalibrationGraph` to `graphInstance`. I'm doing the same thing for my current PR (#288